src/EditorFeatures/CSharpTest/InvertIf/InvertIfTests.cs

```diff
@@ -1037,5 +1037,100 @@ public async Task TestIsExpression()
 @"class C { void M(object o) { [||]if (o is C) { a(); } else { } } }",
 @"class C { void M(object o) { if (o is not C) { } else { a(); } } }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInvertIf)]
+        [WorkItem(43224, "https://github.com/dotnet/roslyn/issues/43224")]
+        public async Task TestEmptyIf()
+        {
+            await TestInRegularAndScriptAsync(
+                @"class C { void M(string s){ [||]if (s == ""a""){}else{ s = ""b""}}}",
+                @"class C { void M(string s){ if (s != ""a""){ s = ""b""}}}");
```

Comment:
tbh, this seems a bit strange.  but i guess it's fine :)

Reply:
which part?

Reply:
taht we are throwing away the empty block.  it feels like if hte user asks to invert, we should just invert.  another feature could potentially say: this block is useless

Reply:
oh...gotcha. I'm going to leave for now because it was my gut that it's how it should work. That said, I'm not really opposed to leaving the empty block if we find that people don't like it being removed